src/pages/Detail/components/DetailArticle/DetailSlider.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Slider from 'react-slick';
+import styled from 'styled-components';
+
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+export default function DetailSlider({ imageUrlList }) {
+  const settings = {
+    dots: true,
+    infinite: true,
+    speed: 500,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+  };
+
+  return (
+    <StyledSlider {...settings}>
+      {imageUrlList &&
+        imageUrlList.map((url, index) => {
```

Comment:
```suggestion
      {imageUrlList?.map((url, index) => {
```
어차피 imageUrlList가 있다는 전제 하에 로직이 작동하니까 조건부 렌더링 없어도 될거 같습니다
혹은 옵셔널 체이닝으로 좀 더 간단히 처리할 수 있겠네요

Reply:
넵 종택님, 해당 조건이 없을시 에러발생해서 옵셔널 체이닝으로 수정하였습니다. 감사합니다!